ex18.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+
+
+def count_days(month: int, year: int):
+    """
+    Function counts the number of days in a specific month
+    :param month: month number
+    :param year: year number
+    :return: number of days in that month
+    """
+
+    # December case
```

Comment:
https://www.w3resource.com/python-exercises/numpy/python-numpy-datetime-exercise-3.php
Look at this solution